trading_calendars/xtks_holidays.py

```diff
@@ -261,6 +262,7 @@ def sunday_to_wednesday(dt):
     Timestamp('2020-09-22'),
     Timestamp('2021-09-23'),
     Timestamp('2022-09-23'),
+    Timestamp('2024-09-23'),
```

Comment:
I know it is in the past and this year the Autumnal Equinox was on Saturday, but could you add it here in the past nonetheless just for completeness?

Reply:
alright! done in [240d0e1](https://github.com/AlpacaTechJP/trading_calendars/pull/70/commits/240d0e1ca617c82475a0286f9e1379bac5c468a0)